task/cerberus-check/0.1/cerberus-check.yaml

```diff
@@ -0,0 +1,77 @@
+---
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: cerberus-check
+  labels:
+    app.kubernetes.io/version: "0.1"
+    app.kubernetes.io/name: "cerberus-check-task"
+    tool: cerberus
+  annotations:
+    tekton.dev/categories: Testing
+    tekton.dev/displayName: cerberus-check
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/platforms: linux/amd64
+    tekton.dev/tags: cerberus, chaos, test
+spec:
+  description: >-
+    The cerberus-check `task`` execute periodic check of a cerberus server and return
+    a OK when cerberus signal is positive.
+
+    Cerberus is a cluster watchdog who return health status of openshift or kubernetes
+    cluster. This task allow you to check this signal as part of your chaos pipeline.
+  params:
+    - name: cerberus_server
+      type: string
+      description: Full URL to the cerberus server
+      default: "http://cerberus:8080"
+    - name: timeout
+      type: string
+      description: Timeout for the healthcheck performed against the cerberus server
+      default: "60"
+    - name: frequency
+      type: string
+      description: Frequency of the healthcheck in case of bad pulse
+      default: "10"
+  steps:
+    - image: "quay.io/startx/fedora:35"
```

Comment:
can we use any lightweight image here? I can see that the following script uses curl and some basic bash scripting

Reply:
corrected with my last commit https://github.com/tektoncd/catalog/pull/1004/files/577a3839e36aaad97f9aec22bc6fc03ae113e237..HEAD